docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  web:
+    build: .
+    ports:
+      - "4000:4000"
+    depends_on:
+      - db
+      - redis
+    volumes:
+       - .:/app
+    environment:
+      - GUARDIAN_KEY=cHvvEICoN0azBi61y6np0nC35e9cObFsH3Yio6LbEtFkwCG7qzqo/qFtVJ+TLTsA
+      - DB_USERNAME=postgres
+      - DB_HOSTNAME=db
+      - DB_PORT=5432
+      - DB_NAME=outbreak_backend_dev
+      - MIX_ENV=dev
+      - REPLACE_OS_VARS=true
+      - SCHEME=http
+      - SECRET_KEY_BASE=cHvvEICoN0azBi61y6np0nC35e9cObFsH3Yio6LbEtFkwCG7qzqo/qFtVJ+TLTsA
```

Comment:
so this is dev only? Seems like these keys shouldn't be checked into version control

Reply:
ahhhh nice catch! these values are made up so no need to rotate them on prod, but no need to have them in this file